[![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7505859.svg)](https://doi.org/10.5281/zenodo.7505859)
# Metagenomics Overview

## Overview of the Cloud-SPAN Metagenomics module

Cloud-SPAN is a collaboration between the [Department of Biology, University of York](https://www.york.ac.uk/biology/) and [The Software Sustainability Institute](https://www.software.ac.uk/) funded by the UKRI innovation scholars award (Project Reference: [MR/V038680/1](https://www.google.com/url?q=https%3A%2F%2Fgtr.ukri.org%2Fprojects%3Fref%3DMR%252FV038680%252F1&sa=D&sntz=1&usg=AFQjCNF0nsozFp-1kvcp0Dgjks6kY8CiCQ)) which trains researchers, and the research software engineers that support them, to run specialised analyses on cloud-based high-performance computing infrastructure

The site infrastructure is based on [The Carpentries](https://carpentries.org/).

This course teaches data analysis for metagenomics projects. It covers how to:
- generate and QC a metagenome assembly
- ‘bin’ the assembly into metagenome assembled genomes (MAGs) also known as bins
- identify the taxonomy of these MAGs.
- calculate diversity metrics and add functional annotation to identify the products of genes identified in the assembled MAGs.
