---
layout: page
title: Frequently Asked Questions
---

Thank you for your interest in participating in this Cloud-SPAN module. Below you will find answers to some frequently asked questions. If the answer to your question doesn’t appear below, please contact [cloud-span-project.york.ac.uk](mailto:cloud-span-project.york.ac.uk) or open an issue in one of [our Github repos](https://github.com/Cloud-SPAN).

* [For Learners](#learners)

## <a id="learners"></a> Learners

### What does this workshop cover?

This course teaches data analysis for metagenomics projects. It covers how to:

generate and QC a metagenome assembly
‘bin’ the assembly into metagenome assembled genomes (MAGs) also known as bins
identify the taxonomy of these MAGs.
calculate diversity metrics and add functional annotation to identify the products of genes identified in the assembled MAGs.


### What experience do you need to have before this workshop? What will you be able to do by the end of the workshop?

This course assumes no prior experience with the tools covered in the workshop but learners are expected to have some familiarity with biological concepts, including the concept of microbiome. Participants should bring their own laptops and plan to participate actively. To get started, follow the directions in the Setup tab to get access to the required software and data for this workshop.

By the end of the module, you will be able to:

- Assemble a genome from long reads and assess the quality of the the assembly.
- Polish this long genome with short reads and generate quality statistics comparing the quality of the unpolished and polished assembly.
- Add taxonomic annotations to the raw reads and identify the abundance of each taxon in the dataset
- Assign the metagenome contigs into bins with each one representing a putative organism/ metagenome assembled genome (MAG).
- Assign gene function to the MAGs identified
- A more detailed list of the learning objectives for each week will be available to you in the course materials

### What are the software, hardware, and connectivity needs for this workshop?

You will need a laptop (not a tablet) and Windows users will also need to install [Git Bash](https://gitforwindows.org). There are no other hardware or software requirements. You will need a stable, strong internet connection.


### Where can I find more information about this workshop?
A full description of this give given here: [Metagenomics overview](https://cloud-span.github.io/metagenomics00-overview/)
